posthog/models/feedback/survey.py

```diff
@@ -267,10 +270,78 @@ def update_response_sampling_limits(sender, instance, **kwargs):
     instance.response_sampling_daily_limits = json.dumps(schedule)
 
 
+def update_question_stable_indices(sender, instance: Survey):
+    """
+    Ensures all questions have a stable_index and maintains max_question_stable_index.
+    This runs before saving a survey, ensuring indices are always computed server-side.
+    Key behaviors:
+    - New surveys: Assigns sequential indices starting from 0
```

Comment:
IMO IDs as integers are always a bad idea unless your code is fully synchronized, what is the change to use a UUID instead